babel.config.js

```diff
@@ -22,6 +22,7 @@ module.exports = function (api) {
           },
         },
       ],
+      "react-native-reanimated/plugin",
```

Comment:
This was missing, I followed https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/getting-started/